File: backend/DataAccess/DaoManual/システム共有/tm_afmsgctrlDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             メッセージコントロールマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afmsgctrlDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afmsgctrl";
		public string ctrlmsgid { get; set; }                     //コントロールメッセージID
		public string ctrlmsgnm { get; set; }                     //コントロールメッセージ名
		public EnumMsgCtrlKbn msgkbn { get; set; }                //メッセージ区分
		public string errormsgid { get; set; }                    //エラーメッセージID
		public string alertmsgid { get; set; }                    //アラートメッセージID
		public string biko { get; set; }                          //備考
    }
}

File: backend/JbdGjsService/Db/DaoManual/システム共有/tm_afmsgctrlDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             メッセージコントロールマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tm_afmsgctrlDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afmsgctrl"
        Public Property ctrlmsgid As String                     'コントロールメッセージID
        Public Property ctrlmsgnm As String                     'コントロールメッセージ名
        Public Property msgkbn As EnumMsgCtrlKbn                'メッセージ区分
        Public Property errormsgid As String                    'エラーメッセージID
        Public Property alertmsgid As String                    'アラートメッセージID
        Public Property biko As String                          '備考
    End Class
End Namespace
